features/conveyancer_adds_title_number_to_deed.feature

```diff
@@ -9,14 +9,18 @@ Acceptance Criteria
   Response code returned for successful and unsuccessful request
   A link to the resource must be returned as part of the response
 
+@us6
 Scenario: Add Multiple Title Numbers
-  Given I have deed data with two title numbers
+  Given I setup a deed with <1> borrowers
+  And I amend "title_number" to "DT567568, DN573253" for borrower <1>
```

Comment:
Thanks for catching this. I've set the borrower number to be optional in the step def. So theres no need to mention it unless amending borrower specific variables.